sdk/armcore/poller_test.go

```diff
@@ -27,10 +27,8 @@ type mockType struct {
 func getPipeline(srv *mock.Server) azcore.Pipeline {
 	return azcore.NewPipeline(
 		srv,
-		azcore.NewTelemetryPolicy(azcore.TelemetryOptions{}),
```

Comment:
Just curious, why was this removed?

Reply:
It's not germane to the test so I removed it.